### DISCUSSION TITLE
What happened to the CDN version?

### DISCUSSION THREAD

**Discussion by Mi113r** [tailwindlabs/tailwindcss#5440] [Help] [answered]
Two hours ago everything was working fine, then some of the styles just stopped working, such as bg-{color}-{int} and the like.

Does anyone have such problems? 
![Screenshot 2021-09-08 142844](https://user-images.githubusercontent.com/6532552/132501492-0271420b-83a2-4e1d-94ca-24639bac0ba2.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

What CDN link are you using?

https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css ?

If yes, is this still an issue? I can see the `bg-indigo-50` color (and others) in there.

<img width="358" alt="image" src="https://user-images.githubusercontent.com/485747/132608873-df799019-5f88-4e9a-bd13-832ae7f0fc19.png">

**Reply by Mi113r** [reply]
yes, i'm use it, but yesterday isn't working. 
Looking today, all good.
